webapp/apps/constants.py

```diff
@@ -1,15 +1,15 @@
 import os
 
-DIAGNOSTIC_TOOLTIP = "Diagnostic table tooltip"
-DIFFERENCE_TOOLTIP = "Difference table tooltip"
-PAYROLL_TOOLTIP = "Payroll info tooltip"
-INCOME_TOOLTIP = "Income info tooltip"
-BASE_TOOLTIP = "Base plan tooltip"
-REFORM_TOOLTIP = "Reform plan tooltip"
-EXPANDED_TOOLTIP = "Expanded income tooltip"
-ADJUSTED_TOOLTIP = "Adjusted income tooltip"
-INCOME_BINS_TOOLTIP = "Income bins tooltip"
-INCOME_DECILES_TOOLTIP = "Income deciles tooltip"
+DIAGNOSTIC_TOOLTIP = "Key variables in the computation of tax liabilities."
+DIFFERENCE_TOOLTIP = "Key variables that highlight the differences between two tax plans."
+PAYROLL_TOOLTIP = " Include employee and employer payroll taxes in output."
+INCOME_TOOLTIP = "Include individual income taxes in output."
+BASE_TOOLTIP = "Current law as defined by default parameters."
+REFORM_TOOLTIP = "The reform proposal as defined by user-provided parameters."
+EXPANDED_TOOLTIP = "The tab variable includes adjusted gross income, non-taxable interest income, taxable investment income excluded from AGI, non-taxable social security benefits, the employer’s share of FICA, and above the line adjustments to AGI on form 1040."
+ADJUSTED_TOOLTIP = "Not yet available."
+INCOME_BINS_TOOLTIP = "Tax units are binned according to the income group to which they belong."
+INCOME_DECILES_TOOLTIP = " Tax units are binned according to the income decile to which they belong."
```

Comment:
This line has the non-ascii quotation character (unicode?) ’ which is causing problems. Changing that back to the ascii ' will fix this.